tests/test_organization.py

```diff
@@ -621,6 +621,21 @@ def test_user_org_post(self):
         self.assert200(rv)
         self.assertEqual(len(rv.json['organizations']), 2)
 
+    def test_user_org_bogus_identifier(self):
+        self.shallow_org_tree()
```

Comment:
@John0Randolph Just FYI these are the type of setUp-type functions that you might wanna try implementing as a [py.test fixture](https://docs.pytest.org/en/latest/fixture.html)